cypress.json

```diff
@@ -0,0 +1,3 @@
+{
+  "baseUrl": "http://localhost:3001"
```

Comment:
I would add the security setting from start.

Reply:
Noted and added.